src/main/java/core/basesyntax/exeptions/InvalidUserAgeException.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.exeptions;
```

Comment:
There is a typo in the package name `exeptions`. It should be `exceptions` to follow standard naming conventions and avoid confusion.